### PR TITLE
AB#94433 Added validation for project start date

### PR DESCRIPTION
**Pull request by MichalSzrederPWC**
### 🛠 Changes being made

Summarize the changes you've made in this pull request.

### 📸 Screenshots (optional)

Insert some screenshots if you made UI changes.

### 🏎 Quality check

- [x] Have you performed local tests?
- [x] Are integration tests passing locally?
- [x] Have you added some unit tests?
- [ ] Have you added some integration tests?
- [x] Have you checked WCAG (included screenshot)

### CODE REVIEW

File: src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs [hunk 1]
```diff
@@ -1,40 +1,43 @@
...
 using HE.Investments.Common.Domain;
+using HE.Investments.Common.Domain.ValueObjects;
 using HE.Investments.Common.Extensions;
 using HE.Investments.Common.Messages;
 using HE.Investments.Common.WWW.Extensions;
 using HE.Investments.Loans.BusinessLogic.Projects.Consts;
 
 namespace HE.Investments.Loans.BusinessLogic.Projects.ValueObjects;
-public class StartDate : ValueObject
+public class StartDate : DateValueObject
```
Comment (jjankowski87) [inline src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs:11]: If you're changing this I suggest to remove dependency to ProjectDate, make sure error messages are correct

**Review by Carq — APPROVED** [review]

File: src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs (hunk 1 above)
Comment (MichalSzrederPWC) [reply src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs:11]: Done,
one question though: in the DateValueObject constructor there is validation to allow only datetimes between year 1900 and 9999, but other constructor do not enforce this validation. Was it deliberate?

**Review by jjankowski87 — APPROVED** [review]

File: src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs (hunk 1 above)
Comment (MichalSzrederPWC) [reply src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs:11]: todo: verify messages with gds and errors page on wiki

Comment (MichalSzrederPWC) [reply src/HE.Investments.Loans.BusinessLogic/Projects/ValueObjects/StartDate.cs:11]: > I don't think so, only reason why there is such value is validation on CRM side - I think it's fine to have min StartDate set to 1900

Removed unnecessary validation for startdate coming from crm as well

**Pull request merged**